tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -365,3 +366,11 @@ Verify Image Can Be Spawned
     Spawn Notebook With Arguments    ${retries}    ${image}    ${size}
     ...    ${spawner_timeout}    ${gpus}    ${refresh}    &{envs}
     End Web Test
+
+Verify Library Version Is Greater Than
+    [Arguments]     ${library}      ${target}
+    ${ver} =  Run Cell And Get Output   !pip show ${library} | grep Version: | awk '{split($0,a); print a[2]}' | awk '{split($0,b,"."); printf "%s.%s.%s", b[1], b[2], b[3]}'
+    ${comparison} =  GTE  ${ver}  ${target}
+    IF  ${comparison}==False
+        FAIL    Library Version Is Smaller Than Expected
```

Comment:
I think the better option will be **Run Keyword And Continue On Failure** otherwise you whole test case will fail. I think you should avoid that

Reply:
I have used **Run Keyword And Continue On Failure** while using this keyword. Even if this keyword fails the whole test case should complete that's why I have used it.